tests/test_bulk_create.py

```diff
@@ -33,6 +34,8 @@ def test_values(self):
         self.assertEqual(1, bulk_create(TestModel, [{'name': 'abc'}]))
         self.assertEqual(1, bulk_create(TestModel, [{'name': 'abc', 'int_field': 2}]))
 
+        self.assertEqual(1, bulk_create(TestModelWithSchema, [{'name': 'abc'}]))
```

Comment:
It is not `InputFormat` test. It doesn't check validation issues.
What does it check? I'd rather add it to `TestSimple` class as separate test method.